mlflow/R/mlflow/R/tracking-client.R

```diff
@@ -1,33 +1,77 @@
-new_mlflow_client <- function(tracking_uri, server_url = NULL) {
```

Comment:
For consistency, a better name would be `mlflow_new_client()`, since all our functions are prefixed with `mlflow_*`.

Reply:
This is an internal constructor and `new_mlflow_client()` is the expected name for the constructor of `mlflow_client` so I think we're good here.